usage-examples/go/atlas-sdk-go/examples/performance/scaling/main.go

```diff
@@ -0,0 +1,199 @@
+// :snippet-start: scale-cluster-programmatically-prod
+// :state-remove-start: copy
+// See entire project at https://github.com/mongodb/atlas-architecture-go-sdk
+// :state-remove-end: [copy]
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"atlas-sdk-go/internal/auth"
+	clusterutils "atlas-sdk-go/internal/clusters"
```

Comment:
Q: should we rename `clusters` to `clusterutils` so we don't have to use an import alias? It's different than how we're using the other internal imports so just had to ask.